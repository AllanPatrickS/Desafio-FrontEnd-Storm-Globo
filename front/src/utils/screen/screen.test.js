import { cleanup } from '@testing-library/react';
import { create } from "react-test-renderer";
import Screen from './screen';

describe('Testing screen', () => {

  it('renders correctly', () => {
    const wrapper = create(<Screen />)
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  afterEach(function () {
    cleanup();
  });

});