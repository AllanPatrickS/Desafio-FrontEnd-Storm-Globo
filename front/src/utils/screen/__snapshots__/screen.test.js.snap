// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing screen renders correctly 1`] = `
Array [
  "0",
  "0",
]
`;
